Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroGraphics/Driver/MicroGraphics.cs

```diff
@@ -46,12 +46,26 @@ public IFont CurrentFont
         /// <summary>
         /// Current color mode
         /// </summary>
-        public ColorType ColorMode => pixelBuffer.ColorMode;
+        public ColorMode ColorMode => pixelBuffer.ColorMode;
 
         /// <summary>
         /// Current rotation used for drawing pixels to the display
         /// </summary>
-        public RotationType Rotation { get; set; } = RotationType.Default;
+        public RotationType Rotation
+        { 
+            get
+            {
+                if(display is IRotatableDisplay {} d) { return d.Rotation; }
+                return _rotation;
+            }
+            set
+            {
+                if (display is IRotatableDisplay {} d) { d.SetRotation(value); }
+                else { _rotation = value; }
```

Comment:
Is MicroGraphics responsible for this, or the DisplayBuffer (when using non-hardware rotation)?

Reply:
Currently the display buffers themselves have no concept of rotation, so MicroGraphics does the translation when any drawing calls are made (set pixel, draw rect, etc.)